h/security/identity.py

```diff
@@ -1,11 +1,20 @@
 """Data classes used to represent authenticated users."""
 
-from dataclasses import dataclass
+from dataclasses import dataclass, field
 from typing import List, Optional, Self
 
 from h.models import AuthClient, Group, User
 
 
+@dataclass
+class LongLivedMembership:
+    """A membership object that isn't connected to SQLAlchemy."""
+
+    group: "LongLivedGroup"
+    user: "LongLivedUser"
```

Comment:
Do we use LongLivedMembership outside identity.user.membership? Maybe we don't need user here.